data/reusables/pages/wildcard-dns-warning.md

```diff
@@ -1,5 +1,5 @@
 {% warning %}
 
-**Warning:** We strongly recommend not using wildcard DNS records, such as `*.example.com`. A wildcard DNS record will allow anyone to host a {% data variables.product.prodname_pages %} site at one of your subdomains even when they are verified. For more information, see "[Verifying your custom domain for {% data variables.product.prodname_pages %}](/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)."
+**Warning:** We strongly recommend not using wildcard DNS records, such as `*.example.com`. A wildcard DNS record will allow anyone to host a {% data variables.product.prodname_pages %} site at one of your subdomains even when they are not verified. For more information, see "[Verifying your custom domain for {% data variables.product.prodname_pages %}](/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)."
```

Comment:
I'm going to check in with our internal team about this. I'm suspicious that the original text might be "correct" but ambiguous. 
 I'm wondering if it should read something like:

> **Warning:** We strongly recommend that you do not use wildcard DNS records, such as `*.example.com`. If you use a wildcard DNS record, this will allow anyone to host a {% data variables.product.prodname_pages %} site at one of your subdomains even if you verify the domain. For more information, see "[Verifying your custom domain for {% data variables.product.prodname_pages %}](/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)."

Reply:
Yes I totally understand what you meant.

Reply:
```suggestion
**Warning:** We strongly recommend not using wildcard DNS records, such as `*.example.com`. These records put you at immediate risk of domain takeovers, including when the domain is verified. For example, verifying `example.com` will prevent someone from using `a.example.com` but not `b.a.example.com` (which is covered by a wildcard DNS record). For more information, see "[Verifying your custom domain for {% data variables.product.prodname_pages %}](/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)."
```

Reply:
Thank you for the contribution @realKarthikNair!

@felicitymay Feel free to adjust my suggestion above. This is true from the engineering side.

Reply:
Thank you for clarifying how this behaves @yoannchaudet 🙇🏻‍♀️ 

With a few minor changes:

```suggestion
**Warning:** We strongly recommend that you do not use wildcard DNS records, such as `*.example.com`. These records put you at an immediate risk of domain takeovers, even you verify the domain. For example, if you verify `example.com` this prevents someone from using `a.example.com` but they could still take over `b.a.example.com` (which is covered by the wildcard DNS record). For more information, see "[Verifying your custom domain for {% data variables.product.prodname_pages %}](/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)."
```

Reply:
@realKarthikNair - I'm going to merge the suggestion above so that we can see what it looks like in the preview. It would be great to know if the revised version makes the behavior clear to you, because you highlighted this confusing area of the documentation.